### PR TITLE
fix(pictures): fix placeholders off center in edge

### PR DESCRIPTION
**Pull request by DanielHabenicht**
image
Now its centered:
image

### CODE REVIEW

**Comment by DanielHabenicht** [comment]
Preview Environment ready at https://40.115.109.25

**Comment by DanielHabenicht** [comment]
Preview Environment ready at https://40.115.109.25

File: Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.scss [hunk 1]
```diff
@@ -60,5 +60,9 @@
     display: flex;
     align-items: center;
     justify-content: center;
+
+    svg {
+      width: 100%;
+    }
```
Comment (mschwrdtnr) [inline Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.scss]: Please check before merging if we really need this

File: Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
   </div>
   <div *ngIf="!imageLoaded" class="image-placeholder" #placeholder>
     <svg height="30" width="100" viewBox="0 0 100 30">
-      <text x="50" y="15" fill="black" font-size="30" text-anchor="middle" dominant-baseline="central">
+      <text x="50" y="25" fill="black" font-size="30" text-anchor="middle">
```
Comment (paule96) [inline Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.html]: maybe okay... Can someone override the SVG height and width? or the text size?

Comment (DanielHabenicht) [reply Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.html]: of course, you could, but what for?

**Comment by DanielHabenicht** [comment]
@paule96 can you have a look at the preview environment in the Edge Browser, as soon as a new comment is added? Please check if the `width: 100%` attribute is really needed.

File: Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.html (hunk 2 above)
Comment (paule96) [reply Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.html]: can SVG use variabels? Then the y and the x of the text should be depend on the size of the svg and on the text size.

Comment (DanielHabenicht) [reply Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.html]: lets discuss this in person.

File: Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.scss (hunk 1 above)
Comment (DanielHabenicht) [reply Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.scss]: Yep, this is a problem in with Edge 42.17134.1.0 (Microsoft EdgeHTML 17.17134)

File: Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.html (hunk 2 above)
Comment (DanielHabenicht) [reply Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.html]: ... and there was a reasone for this beeing SVG, scalability :)

**Comment by DanielHabenicht** [comment]
Preview Environment ready at https://40.115.109.25

**Pull request merged**